### ISSUE TITLE
It's impossible to write XML without namespaces

### ISSUE THREAD

**Issue by mvysny** [FasterXML/jackson-dataformat-xml#375]
I'm trying to write XML without namespace support, to avoid https://github.com/FasterXML/woodstox/issues/99 . However, the following code fails:
```
        val mapper: XmlMapper = XmlMapper().apply {
            enable(SerializationFeature.INDENT_OUTPUT)
            enable(ToXmlGenerator.Feature.WRITE_XML_DECLARATION)
            setSerializationInclusion(JsonInclude.Include.NON_NULL)
            factory.xmlOutputFactory.setProperty(XMLInputFactory.IS_NAMESPACE_AWARE, false)
            factory.xmlOutputFactory.setProperty(WstxOutputProperties.P_USE_DOUBLE_QUOTES_IN_XML_DECL, true)
//            factory.xmlOutputFactory.setProperty(XMLOutputFactory.IS_REPAIRING_NAMESPACES, false)
        }
mapper.writeValue(file, this)
```
throws:

```
java.lang.IllegalArgumentException: Can not set default namespace for non-namespace writer.

	at com.ctc.wstx.sw.BaseStreamWriter.reportIllegalArg(BaseStreamWriter.java:1672)
	at com.ctc.wstx.sw.NonNsStreamWriter.setDefaultNamespace(NonNsStreamWriter.java:102)
	at com.fasterxml.jackson.dataformat.xml.XmlFactory._initializeXmlWriter(XmlFactory.java:643)
	at com.fasterxml.jackson.dataformat.xml.XmlFactory.createGenerator(XmlFactory.java:516)
	at com.fasterxml.jackson.dataformat.xml.XmlMapper.writeValue(XmlMapper.java:236)
```

**Comment by cowtowncoder** [comment]
Right, non-namespace mode is not supported; do not try to use that.

**Comment by cowtowncoder** [comment]
Since #395 is now fixed (moved from Woodstox), will be in 2.10.5 / 2.11.1, closing this one.
